custom/enikshay/tasks.py

```diff
@@ -76,31 +91,59 @@ class EpisodeUpdater(object):
     This is applicable to various enikshay domains. The domain can be specified in __init__ method
     """
 
-    def __init__(self, domain):
+    def __init__(self, domain, task_id=None):
         self.domain = domain
+        self.task_id = task_id
         self.updaters = [
             EpisodeAdherenceUpdate,
```

Comment:
I don't have a lot of context here, but wondering if it would be possible to split this up based on these updaters? Like could you have 3 tasks with one of these?